git/git.go

```diff
@@ -80,6 +96,19 @@ func ResolveRef(ref string) (*Ref, error) {
 	return fullref, nil
 }
 
+func ResolveRefs(refnames []string) ([]*Ref, error) {
+	refs := make([]*Ref, len(refnames))
+	for i, name := range refnames {
+		ref, err := ResolveRef(name)
+		if err != nil {
+			return refs, err
+		}
+
+		refs[i] = ref
+	}
+	return refs, nil
+}
```

Comment:
Haha, tale as old as time: 1. criticise code 2. realise you wrote it 3. Facepalm xD

Still, if it's going to be its own function it might as well improve things too. Clutching at the shreds of my dignity... ;D